### PR TITLE
Distribution

### PR DESCRIPTION
**Pull request by DimEvil**

### CODE REVIEW

File: src/convert_to_dwc.R [hunk 1]
```diff
@@ -43,13 +55,15 @@ DwcTaxonListEU %<>%
...
-  gbif_species_name_match(name_col = "scientificName", 
+  gbif_species_name_match(name_col = "scientificNameLocal", 
                           gbif_terms = c('usageKey', 'scientificName', 'rank', 
                                          'order', 'matchType','phylum', 
                                          'kingdom', 'genus','class',
                                          'confidence', 'synonym', 'status',
-                                         'family'))
-DwcTaxonListLocal %<>% 
+                                         'family'))%>%
+  mutate(taxonID = paste(usageKey, vdigest (scientificName, algo="md5"), sep=":"))
```
Comment (stijnvanhoey) [inline src/convert_to_dwc.R:64]: just curious, what is this step doing?

Comment (DimEvil) [reply src/convert_to_dwc.R:64]: Create TaxonID

Comment (stijnvanhoey) [reply src/convert_to_dwc.R:64]: so you create a unique identifier yourself using the md5 algorithm?

Comment (DimEvil) [reply src/convert_to_dwc.R:64]: plus GBIF usageKey

**Comment by stijnvanhoey** [comment]
all good to me!

**Review by stijnvanhoey — APPROVED** [review]

**Pull request merged**